pkgs/development/python-modules/ormsgpack/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cargo,
+  rustPlatform,
+  rustc,
+  pyinstaller,
+
+  # dependencies
+  msgpack,
+
+  # testing
+  pydantic,
+  pytestCheckHook,
+  python-dateutil,
+  pytz,
+}:
+
+buildPythonPackage rec {
+  pname = "ormsgpack";
+  version = "1.9.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "aviramha";
+    repo = "ormsgpack";
+    rev = version;
```

Comment:
```suggestion
    tag = version;
```

Reply:
Thank you. I knew your eye for detail would likely find something! Done.